### PR TITLE
[build / ci] Add py38 and Django 3.0 to the test matrix and cleanup

### PR DESCRIPTION
**Pull request by Djailla**
Add support for new python / Django versions.

### CODE REVIEW

**Comment by trbs** [comment]
Please fix the typo in the comment/review above.
Otherwise LGTM.

What do you think about dropping `py{34,35,36,37}-flake8` ? 
Thinking those don't add that much... if anything having `pypy-flake8` would make more sense I guess ;-)

**Comment by trbs** [comment]
With the last change all the different `flake8` builds disappeared :(

**Comment by Djailla** [comment]
@trbs yes fixed with next commit. Trying to code on mobile is a painful experience ;)

**Comment by trbs** [comment]
haha no problem :-)
I feel your pain, I'll be traveling myself a lot as well these weeks so no need to rush ;-)

**Comment by Djailla** [comment]
It worked with last commit. All good !

**Comment by trbs** [comment]
I just see `flake8` and am missing `py27-flake8` and `py38-flake8` ?

**Comment by Djailla** [comment]
It’s on purpose. Flake8 will provide the same output whatever python version it’s started with.

**Comment by Djailla** [comment]
Hello @trbs 

Any update about the failing CI ?

**Comment by Djailla** [comment]
@trbs up ?

**Comment by trbs** [comment]
> Hello @trbs
> 
> Any update about the failing CI ?

For now I disabled the failing test since I cannot fix it at the moment. Don't have time to further investigate, so better to loose a couple of tests then have the entire suite failing ;-)

File: tox.ini [hunk 1]
```diff
@@ -8,16 +8,18 @@ envlist =
...
+    flake8
+    py{27,34,35,36,37,py,py3}-dj111
+    py{34,35,36,37,py3}-dj20
+    py{35,36,37,py3}-dj21
+    py{35,36,37,py3}-dj22
+    py{36,37,38,py3}-dj30
+    py{36,37,38,py3}-djmaster
     py37-dj22-postgres
+    py38-dj30-postgres
+    pt{37,38}-djmaster-postgres
```
Comment (trbs) [inline tox.ini]: typo ?

**Comment by trbs** [comment]
Wondering why it fails on `py38` here and why it does not in the master builds.

On a related note... `keyczar` should really be deprecated and removed... so I don't see it as too much of a blocker. (eg: skipping the `keyczar` on `py38` or removing them all together would be fine with me as well since nobody should be using `keyczar` anyways)

That said would be good to know why it's failing here and not on the master builds :-)

**Comment by Djailla** [comment]
@trbs : it also fails with `py38` over `master`

**Comment by Djailla** [comment]
@trbs now it's failing with `py34` :'(

**Comment by trbs** [comment]
i tried to fix this by adding your changed by hand in master...

https://github.com/django-extensions/django-extensions/commit/0ad61ae76b121344ccd2ad4bbf4af4e4b5b8f70e
https://travis-ci.org/django-extensions/django-extensions?utm_medium=notification&utm_source=github_status

**Comment by Djailla** [comment]
Perfect, thank you !

I'll close this PR

**Comment by trbs** [comment]
Thanks for your help !

**Pull request closed without merging**